test/macros.jl

```diff
@@ -1,6 +1,8 @@
-@testset "macros" begin
+module macros
```

Comment:
Made this change for the following reasons (along with the corresponding change on runtests.jl):
- To prevent name collisions with other test files
- To enable `test/macros.jl` to be executed as a standalone script from the command line

Having said that, this creates a different style from other test files, so if you don't like this, I'd be happy to revert it.

Reply:
I don't really mind this change.  But if you first `include("test/utils.jl")` most (all?) of the non-IR test/blah.jl should be able to be included individually at the REPL? This is how I run tests though it's admittedly slightly clunky.